source/main.js

```diff
@@ -168,6 +171,15 @@ app.on('activate', function () {
   zettlr.openWindow()
 })
 
+/**
+ * Load Vue developer extension
+ */
+app.whenReady().then(() => {
```

Comment:
Can we put this in the `app.on('ready')`-event listener? This would ensure that this is called BEFORE the application is actually instantiated and keep the logic in one place

Reply:
I didn't really understood what I did there, just followed the official documentation ;-) I just googled a bit and apparently the `whenReady` is preferred over `on(ready)`, see https://github.com/electron/electron/pull/21972. I've now changed on(ready) to whenReady and merge both methods.

Reply:
I did not realize they had this change, but then Electron sometimes fails at communication. However, if it's now merged, then perfect! :)